requirements.txt

```diff
@@ -1,13 +1,16 @@
 altair==4.2.0
-#required for toolbox but gives issues on Mac when not installed first
-chardet==4.0.0
```

Comment:
See also the readme, but on Mac I need to install `chardet` separately for some magical reason

Reply:
Yeah sorry, I meant to add a note to the README about this. 

Can I ask, what error do you get? `chardet` is needed by the `hdx` packages so I'm wondering if we should loop in Mike.

Reply:
Ofc! 
`ERROR: Failed building wheel for cchardet
..
    clang: error: no such file or directory: 'src/ext/uchardet/src/CharDistribution.cpp'
    clang: error: no input files
    error: command '/usr/bin/clang' failed with exit code 1
`
Which is a known issue for Mac see [here](https://github.com/PyYoshi/cChardet/issues/37) but no clue why.. Just tested and Seth has the same issue.

Reply:
Ah okay, thanks for testing with Seth also. 

> . When just doing pip install aa-toolbox it does work fine even though it seems the requirements.txt for chardet and cchardet in toolbox are the same for version 3.1.0 and 0.4.0dev9

I'm not totally sure what you mean, but note that when you install `aa-toolbox` it won't install what's in `requirements.txt`, it will use the unpinned (so probably latest) ones from `setup.cfg`. 

Would it somehow help to ask Mike to pin to a specific `chardet` version?